cleanrepo/CleanRepo/Program.cs

```diff
@@ -24,10 +24,22 @@ class Program
     static void Main(string[] args)
     {
 #if DEBUG
+        // TODO: Consider using launchSettings.json with command line args instead:
```

Comment:
With the latest changes in the script, are these necessary?

I think they're OK, but I'd like Genevieve's thoughts on any concerns with order, or running multiple commands in a single exe.

Reply:
The debug arguments are useful when debugging during development.

There are other changes (optimizations) that could be made if we're going to run multiple options at once - for example, only look for the docfx.json file once, or only obtain all the Markdown files once. If you want to merge it as-is, I can make those optimizations later.

Reply:
I noticed the potential for those optimizations too but didn't want to make too many changes without getting a nod of approval. Would you prefer to make those optimizations, or do you want me to?

Reply:
Also, if you prefer, I could just undo the changes to _Program.cs_ as they're not needed for this effort. Just let me know.